.travis.yml

```diff
@@ -1,17 +1,21 @@
 language: rust
+rust: nightly
```

Comment:
It would be nice to also test on 1.0.0.  This can be done easily with a list.  For example, https://github.com/posborne/rust-sysfs-gpio/commit/f080e9a303e1a9ea0e1a63103e7b483c30964cd1